geoportailv3/static/js/share/shorturl.html

```diff
@@ -0,0 +1,10 @@
+  <form class="form-horizontal">
+    <div class="col-md-12">
+    <div class="form-group">
+      <label translate class="col-sm-3 control-label">Short Url</label>
+      <div class="col-sm-9">
+        <input onClick="this.select()" type="text" class="form-control" name="shorturl" ng-model="ctrl.url">
```

Comment:
This is pure javascript. `this` refers to the input. `select` selects all the text in this input.
http://jsfiddle.net/u1wg2eku/